src/components/product/preview-card.svelte

```diff
@@ -0,0 +1,105 @@
+<script>
+  import Card from 'ui/card.svelte';
+  import Button from 'ui/button.svelte';
+  import Labeled from 'ui/labeled.svelte';
+  import RadioGroup from 'ui/radio-group.svelte';
+  import RadioChipGroup from 'ui/radio-chip-group.svelte';
+  import getBackground from '@/utils/optimal-color.js';
+  import { API_HOST } from '@/constants/env.js';
+
+  export let product;
+  let selectedIndex = 0;
+  $: selectedVariety = product && product.varieties[selectedIndex];
+  $: imageUrl = (
+    selectedVariety && selectedVariety.images && selectedVariety.images.length > 0 ?
+      API_HOST + selectedVariety.images[0]
+    : '/images/create-product/placeholder.svg'
+  );
+  $: colors = product && product.varieties.map(x => x.color).filter(x => x != null);
+  $: sizes = selectedVariety && Object.keys(selectedVariety.sizes);
+  let altText = '';
+  if (product) {
+    altText = 'Product image: ';
+    if (product.type) {
+      altText += `"${product.name}" ${product.type}`;
+    } else {
+      altText += name;
```

Comment:
I don't see this `name` being defined anywhere
```suggestion
      altText += product.name;
```